packages/jackett/build.sh

```diff
@@ -0,0 +1,44 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/jackett/jackett"
+TERMUX_PKG_DESCRIPTION="API Support for your favorite torrent trackers"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.22.1428"
+TERMUX_PKG_SRCURL="https://github.com/Jackett/Jackett/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256="71caabacc957e0b98e3f425b76099000ac80dacf22077ffcba5f012fb1bb12f1"
+TERMUX_PKG_BUILD_DEPENDS="aspnetcore-targeting-pack-8.0, dotnet-targeting-pack-8.0"
+TERMUX_PKG_DEPENDS="aspnetcore-runtime-8.0, dotnet-host, dotnet-runtime-8.0"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_SERVICE_SCRIPT=("jackett" 'exec ${TERMUX_PREFIX}/bin/jackett --DataFolder ${TERMUX_ANDROID_HOME}/.config/jackett 2>&1')
```

Comment:
Please replace $TERMUX_PREFIX with just $PREFIX and ${TERMUX_ANDROID_HOME} with $HOME. $TERMUX_PREFIX and $TERMUX_ANDROID_HOME are only available during build, not during runtime on device.

Reply:
You haven't fix these

Reply:
Right because they aren't being resolved at build time.
You can either double quote the second string so variables get resolved in it
```suggestion
TERMUX_PKG_SERVICE_SCRIPT=("jackett" "exec ${TERMUX_PREFIX}/bin/jackett --DataFolder ${TERMUX_ANDROID_HOME}/.config/jackett 2>&1")
```
Or you can use the runtime variable names Grimler mentioned above.
```suggestion
TERMUX_PKG_SERVICE_SCRIPT=("jackett" 'exec ${PREFIX}/bin/jackett --DataFolder ${HOME}/.config/jackett 2>&1')
```
Either one should resolve the problem.

Reply:
Well either one is correct.
It just depends if you want to resolve the paths at build time or runtime.
Resolving them at build time (option 1) is probably preferable for a service script.